sklearn/calibration.py

```diff
@@ -712,10 +728,11 @@ class _CalibratedClassifier:
     classes : array-like of shape (n_classes,)
```

Comment:
The docstring of the `calibrators` parameter needs to be updated since the number of calibrators depend differently on whether the calibrator class can support multiclass or binary logits as inputs.